src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -62,54 +81,61 @@ public ArrebolController(Properties properties) {
         this.queue = new TaskQueue(queueId, queueName);
 
         int poolId = 1;
-        WorkerPool pool = createPool(arrebolProperties, poolId);
+        WorkerPool pool = createPool(configuration, poolId);
 
         //create the scheduler bind the pieces together
         FifoSchedulerPolicy policy = new FifoSchedulerPolicy();
         this.scheduler = new DefaultScheduler(queue, pool, policy);
 
-        this.jobPool = Collections.synchronizedMap(new HashMap<String,  Job>());
+        this.jobPool = Collections.synchronizedMap(new HashMap<String, Job>());
         this.jobDatabaseCommitter = new Timer(true);
+        this.jobStateMonitor = new Timer(true);
     }
 
-    private static final String RAW_TYPE = "raw";
-    private static final String DOCKER_TYPE = "docker";
+    private void certifyConfigurationProperties() throws Exception {
+        final String verifyMsg = " Please, verify your configuration file.";
+        final String imageIdMsg =
+            "Docker Image ID configuration property wrong or missing." + verifyMsg;
+        final String poolTypeMsg =
+            "Worker Pool Type configuration property wrong or missing." + verifyMsg;
+        final String resourceAddressesMsg =
+            "Docker Image ID configuration property wrong or missing." + verifyMsg;
+        final String workerPoolSizeMsg =
+            "Docker Image ID configuration property wrong or missing." + verifyMsg;
+
+        String imageId = this.configuration.getImageId();
+        String poolType = this.configuration.getPoolType();
+        List<String> resourceAddresses = this.configuration.getResourceAddresses();
+        Integer workerPoolSize = this.configuration.getWorkerPoolSize();
+
```

Comment:
it seems all exceptions below are logged twice (here and in the constructor)

Reply:
Yes, I removed the error logs.